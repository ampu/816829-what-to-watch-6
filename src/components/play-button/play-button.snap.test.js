import React from 'react';

import {render} from '@testing-library/react';

import PlayButton from './play-button';

describe(`PlayButton component`, () => {
  it(`Should render according to the snapshot`, () => {
    const {container} = render(<PlayButton/>);
    expect(container).toMatchSnapshot();
  });
});
