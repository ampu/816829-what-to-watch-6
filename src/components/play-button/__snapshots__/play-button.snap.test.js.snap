// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayButton component Should render according to the snapshot 1`] = `
<div>
  <button
    type="button"
  >
    <svg
      height="19"
      viewBox="0 0 19 19"
      width="19"
    >
      <use
        xlink:href="#play-s"
      />
    </svg>
    <span>
      Play
    </span>
  </button>
</div>
`;
